src/Views/Timeline.vala

```diff
@@ -114,13 +134,24 @@ public class Tootle.Views.Timeline : Views.Abstract {
         }
 
         var msg = new Soup.Message ("GET", get_url ());
+        if(accounts.currentInstance.version.ascii_casecmp ("3.0.0")>=0 && this.timeline=="direct"){
+            msg = new Soup.Message ("GET", get_url_status ());
+        }
         network.inject (msg, Network.INJECT_TOKEN);
         network.queue (msg, (sess, mess) => {
+                var mensaje=msg;
```

Comment:
I am not sure if it is used... Is it or should it be?

Reply:
Yes, sorry I'm going to remove